plugins/templates/labhub/errors/not-eligible.jinja2.md

```diff
@@ -1,3 +1,4 @@
 - You must be a member of the {{ organization }} org to be assigned an issue. If you are not a member yet, just type Hello World and corobo will invite you.
 - A newcomer cannot be assigned to an issue with a difficulty level higher than newcomer or low difficulty.
 - A newcomer cannot be assigned to unlabelled issues.
+- corobo cannot assign issues labelled with 'initiatives/gci'. Only GCI participants can be assigned these issues.
```

Comment:
This says only GCI participants can be assigned these issues.
However that is not implemented in this patch.